docs/menu.rb

```diff
@@ -0,0 +1,35 @@
+class Menu  
+
+    attr_reader :order
+
+    def initialize
+       @list = { 'Latte' => 2.90, 'Cappuccino' => 2.80, 'Cake' => 3.50, 'Sandwich' => 4.00 }    
+
+       @order = []
+    end  
+
+    def list_menu   
+        @list.each do |key, value|
```

Comment:
Logical code to display the menu. SRP well used.

Reply:
Key and Value aren't very descriptive of what they represent. It may be easier for other people to read if you call them item and price or something more indicative of what they represent.